language/move-prover/interpreter-testsuite/tests/concrete_check_testsuite.rs

```diff
@@ -1,44 +1,32 @@
 // Copyright (c) The Diem Core Contributors
 // SPDX-License-Identifier: Apache-2.0
 
-use anyhow::Result;
 use std::{env, path::Path};
 
-use move_prover_test_utils::{
-    baseline_test::{diff, verify_or_update_baseline},
-    read_bool_env_var,
-};
+use move_prover_test_utils::{baseline_test::verify_or_update_baseline, read_bool_env_var};
 use move_unit_test::UnitTestingConfig;
 
 const BASELINE_EXTENSION: &str = "exp";
 
-fn run_unit_test(path: &Path, use_stackless_vm: bool) -> Result<String> {
+fn test_runner(path: &Path) -> datatest_stable::Result<()> {
+    env::set_var("NO_COLOR", "1");
+
     let config = UnitTestingConfig {
         instruction_execution_bound: 5000,
         filter: None,
         num_threads: 1,
         source_files: vec![path.to_str().unwrap().to_owned()],
-        use_stackless_vm,
+        use_stackless_vm: true,
```

Comment:
`use_stackless_vm` doesn't seem quite right with this change -- it makes me think of only using the stackless VM for execution, and less about comparing execution results between the two. 

What would you think of changing the name of this to something like `compare_against_stackless_vm`, `check_stackless_vm`, or something else along these lines?

Reply:
Right! It was originally named to represent only using the stackless VM but later I changed its meaning without the name :(. I'll rename it to `check_stackless_vm`.